pkg/servicemanager/manager.go

```diff
@@ -52,7 +53,7 @@ func (m *ServiceManager) AddService(s Service) error {
 	return nil
 }
 
-func (m *ServiceManager) Run(ctx context.Context, ready chan<- struct{}, stopped chan<- struct{}) error {
+func (m *ServiceManager) Run(ctx context.Context, ready chan<- struct{}, stopped chan<- struct{}, startLog *startuplogger.StartupLogger) error {
```

Comment:
`*startuplogger.StartupLogger` could be a field of `ServiceManager`. Function params here are service specific, logger is "global"

Reply:
changed